test/document/document_test.ts

```diff
@@ -615,4 +615,56 @@ describe('Document', function () {
     });
     assert.equal('{"data":{"null":null}}', doc.toSortedJSON());
   });
+
+  it('change paths test', async function () {
+    const doc = Document.create('test-col', 'test-doc');
+    await new Promise((resolve) => setTimeout(resolve, 0));
```

Comment:
just for my curiosity, could you explain the reason why this promise code was added?

Reply:
We can subscribe to events fired on the document using `document.subscribe`. In the subscribe, the stream inside the document is initialized asynchronously, so I put this promise code to wait for it.

It's hard for the user to notice this situation, so I think we'll have to find out if we can turn the event stream initialization synchronously.

Reply:
Aha I see! Thank you for explanation!

Reply:
Aha I see! Thank you for explanation!